tests/devapps/XForms/XForms.Android/Properties/AndroidManifest.xml

```diff
@@ -1,16 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:installLocation="auto" package="com.Microsoft.XFormsDroid.MSAL">
-	<uses-sdk android:minSdkVersion="15" />
-	<uses-permission android:name="android.permission.INTERNET" />
-	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-	<application android:label="XForms.Android">
-		<activity android:name="microsoft.identity.client.BrowserTabActivity">
-			<intent-filter>
-				<action android:name="android.intent.action.VIEW" />
-				<category android:name="android.intent.category.DEFAULT" />
-				<category android:name="android.intent.category.BROWSABLE" />
-				<data android:scheme="msale3b9ad76-9763-4827-b088-80c7a7888f79" android:host="auth" />
-			</intent-filter>
-		</activity>
-	</application>
+  <uses-sdk android:minSdkVersion="15" />
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <application android:label="XForms.Android">
+    <activity android:name="microsoft.identity.client.BrowserTabActivity">
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <!--<data android:scheme="msauth-5a434691-ccb2-4fd1-b97b-b64bcfbc03fc" android:host="com.microsoft.identity.client.sample" />-->
```

Comment:
>--<dat [](start = 10, length = 6)

remove?

Reply:
this is needed for system browser...trying to make it easier/faster to test in the future. maybe I should add a comment?

Reply:
@henrik-me I added a comment so it's clearer.